src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -392,17 +396,106 @@ public object VisitTable(TableExpression tableExpression, SearchOptions context)
 
                     _includeCtes.Add(TableExpressionName(_tableExpressionCounter));
                     break;
+                case TableExpressionKind.RevInclude:
```

Comment:
The generated SQL is not indented properly, does not follow formatting conventions.